lib/cli/transaction.rs

```diff
@@ -90,6 +94,31 @@ pub fn make_transaction(
     crate::output_transaction(output, &transaction).map_err(CliError::from)
 }
 
+/// Creates a [`Transaction`] and sends it to the network for execution.
+///
+/// `rpc_id_str` is the RPC ID to use for this request.
+/// `node_address` is the address of the node to send the request to.
+/// `verbosity_level` is the level of verbosity to use when outputting the response.
+pub async fn put_transaction(
+    rpc_id_str: &str,
+    node_address: &str,
+    verbosity_level: u64,
+    builder_params: TransactionBuilderParams<'_>,
+    transaction_params: TransactionStrParams<'_>,
+) -> Result<SuccessResponse<PutTransactionResult>, CliError> {
+    let rpc_id = parse::rpc_id(rpc_id_str);
+    let verbosity_level = parse::verbosity(verbosity_level);
+    let transaction = create_transaction(builder_params, transaction_params, false)?;
+    crate::put_transaction(
```

Comment:
I'm not a huge fan of path qualified calls. Out of curiosity - why not use an alias for `put_transaction` and import it on top of the file?